settings.sh

```diff
@@ -133,6 +133,7 @@ export QUESTA_PATH=$(dirname $(which vsim))
 # Add GCC and Spike to path
 export SW_ROOT=${ROOT_DIR}/sw
 export BOOTROM_COE=${SW_ROOT}/bootrom.coe
+export RV_TOOLCHAIN_PATH=/opt/rv32/bin/riscv32-unknown-elf-
```

Comment:
Remove __absolute paths__ like this, since every one might have a different installation directory, e.g. lack the permission to install in root directories. 

We had agreed to just assume RISC-V bins would be in PATH

Reply:
I think we should leave the ability to define whatever location for the compiler. By default, we can have the riscv32-unknown-elf- prefix in the settings.sh, assuming it  to be in path by default. But, if it is not the case, one can still easily use its own toolchain, even if its not in path

Reply:
Prefix and location of the binaries are two different issues.
- Location should never be hardcoded, for the reasons above
- Prefix can be common, and can be somewhere in a common Makefile, I would not put it in the top settings script, which is more general (e.g. it should not configure builds for both SoC and host, xilinx and verilator, etc)

On the other hand, we must decide if to anticipate or postpone the difference between bare-metal code, OpenSBI payloads and Linux ELFs, therefore changing the prefix.

Reply:
Such a difference is going to emerge sooner or later and we should discuss  about it asap.
In fact, we could choose for a variable to modify depending on where your toolchain is and what toolcha it is (is it custom for custom extensions? is it 64 or 32 bits? Unknown or Linux?). Though this should be discussed in a specific meeting

Reply:
New item on the chart 🤣 ?

Reply:
I would first discuss it in a meeting. Then it can be associated with CVA6 integration task or Linux integration task.